book_source/03_topical_pages/09_standalone_tools.Rmd

```diff
@@ -159,83 +159,220 @@ Processing functions currently available are,
+This module is accesible using the R function `remote_process` which uses the Python package "RpTools" (located at `data.remote/inst/RpTools`) for downloading and processing data. RpTools has a function named `rp_control` which controls two other functions,
+
+1. `get_remote_data` which controls the scripts which are used for downloading data from the source. For example, `gee2pecan_s2` downloads bands from Sentinel 2 using GEE.
+
+2. `process_remote_data` which controls the scripts responsible for processing the raw data. For example, `bands2lai_snap` uses the downloaded bands to compute LAI using the SNAP algorithm.
+
+![Workflow of the module](03_topical_pages/11_images/remotemodule.png)
+
+
+### Configuring `remote_process`
+
+`remote_process` is configured using remote data tags in the `pecan.xml`. The required tags are described below,
+
+```xml
+  <remotedata>
+  <out_get_data>...</out_get_data>
+  <source>...</source>
+  <collection>...</collection>
+  <scale>...</scale>
+  <projection>...</projection>
+  <qc>...</qc>
+  <algorithm>...</algorithm>
+  <credfile>...</credfile>
+  <out_process_data>...</out_process_data>
+  <overwrite>...</overwrite>
+  </remotedata>
+```
+
+
+* `out_get_data`: (required) type of raw output requested, e.g, bands, smap
+* `source`: (required) source of remote data, e.g., gee or appeears 
+* `collection`: (required) dataset or product name as it is provided on the source, e.g. "COPERNICUS/S2_SR" for gee or "SPL3SMP_E.003" for appeears
+* `scale`: (optional) pixel resolution required for some gee collections, recommended to use 10 for Sentinel 2 **scale** Information about how GEE handles scale can be found out [here](https://developers.google.com/earth-engine/scale)
+* `projection`: (optional) type of projection. Only required for appeears polygon AOI type
+* `qc`: (optional) quality control parameter, required for some gee collections
+* `overwrite`: (optional) if TRUE database checks will be skipped and existing data of same type will be replaced entirely. When processed data is requested, the raw data required   for creating it will also be replaced. By default FALSE
+
+These tags are only required if processed data is requested:
+
+* `out_process_data`: (optional) type of processed output requested, e.g, lai
+* `algorithm`: (optional) algorithm used for processing data, currently only SNAP is implemented to estimate LAI from Sentinel-2 bands
+* `credfile`: (optional) absolute path to JSON file containing Earthdata username and password, only required for AppEEARS
+
+Additional information are taken from the registration files located at pecan/modules/data.remote/inst/registration, each source has its own registration file. This is so because there isn't a standardized way to retrieve all image collections from GEE and each image collections may require its own way of performing quality checks, etc wheras all of the products available on AppEEARS can be retrieved using its API.
```

Comment:
```suggestion
Additional information are taken from the registration files located at pecan/modules/data.remote/inst/registration, each source has its own registration file. This is so because there isn't a standardized way to retrieve all image collections from GEE and each image collections may require its own way of performing quality checks, etc whereas all of the products available on AppEEARS can be retrieved using its API in a standardized way.
```
Thanks, could you also link `pecan/modules/data.remote/inst/registration`